feeadjuster/test_feeadjuster.py

```diff
@@ -71,11 +85,12 @@ def test_feeadjuster_adjusts(node_factory):
     chan_B = l2.rpc.listpeers(l3.info["id"])["peers"][0]["channels"][0]
     scid_A = chan_A["short_channel_id"]
     scid_B = chan_B["short_channel_id"]
-    l2_scids = [scid_A, scid_B]
+    nodes = [l1, l2, l3]
+    scids = [scid_A, scid_B]
 
     # Fees don't get updated until there is a forwarding event!
     assert all([get_chan_fees(l2, scid) == (base_fee, ppm_fee)
-                for scid in l2_scids])
+                for scid in scids])
```

Comment:
Huh, that does not hold anymore..

Reply:
And yet the test pass

Reply:
What do you mean? We just renamed from l2_scids to just scids, why shouldn't it work?

Reply:
This is not the renaming that should not work, but the assertion. See the comment: now the fees *are* updated before the forwarding event (at init). I think it's just racy (removing it in my followup).

Reply:
Hm right. Let's check that tomorrow and do another round. Also the manual calling of 'feeadjust' in the imbalance testcase does not really make sense if it's called by init. We should add a dedicated testcase for this.

Reply:
So while tracking why https://github.com/lightningd/plugins/pull/160/files#diff-1da741a1bd87184f52856debf2dcbc39c507c574e03f44f689e652606c92d933R175-R177 wasn't superfluous as i thought at first, i figured i'm wrong here as the `init`'s `feeadjust` will be called without any channel :)

Reply:
> We should add a dedicated testcase for this.

Héhé, agreed: https://github.com/darosior/plugins/commit/884d7e837d84eb14aeced663fc549b5a2e9fe650 :D